ui/app/components/project-config-variables/list-item.ts

```diff
@@ -83,6 +94,7 @@ export default class ProjectConfigVariablesListItemComponent extends Component<V
   cancelEdit(): void {
     this.isCreating = false;
     this.isEditing = false;
-    this.variable = this.initialVariable;
+    this.changeset?.unexecute();
+    // this.variable = this.initialVariable;
```

Comment:
Similarly, do we need to execute/unexecute the changeset? Or cold we just throw it away at this point?